Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -360,15 +370,17 @@ void EncodeNestedFieldValue(pb_ostream_t* stream,
   // did the first time. (Use an initializer rather than setting fields
   // individually like nanopb does. This gives us a *chance* of noticing if
   // nanopb adds new fields.)
-  substream = {stream->callback, stream->state, /*max_size=*/size,
-               /*bytes_written=*/0, /*errmsg=*/NULL};
+  pb_ostream_t raw_writing_substream = {stream_->callback, stream_->state,
+                                        /*max_size=*/size,
+                                        /*bytes_written=*/0, /*errmsg=*/NULL};
```

Comment:
Shouldn't we prefer `nullptr` to `NULL`?

Reply:
Done.